modules/candidate_parameters/php/NDB_Form_candidate_parameters.class.inc

```diff
@@ -48,39 +48,44 @@ class NDB_Form_candidate_parameters extends NDB_Form
 
         $candidate =& Candidate::singleton($this->identifier);
 
+        $this->tpl_data['candidate_parameter_edit'] = $user->hasPermission('candidate_parameter_edit');
+
         // check user permissions
-        return ($user->hasPermission('candidate_parameter_edit')  
-            && $user->getData('CenterID') == $candidate->getData('CenterID'));  
+        return ($user->getData('CenterID') == '1' ||
```

Comment:
CenterID 1 is, by convention (and in the default schema), the DCC, which isn't a real site that collects data, but just coordinates it.

I haven't looked at this code so I don't know if that warrants special treatment here, but that's generally the reason for comparisons to CenterID == 1 (or != 1) in queries. It looks like the right check to do here might be the access_all_sites permission, not the CenterID=1, though?